app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  has_many :posts
```

Comment:
- [ ] Great effort working on your project, according to the ERD diagram, user `has_many likes and comments` too